sparkles/tests/test_review.py

```diff
@@ -26,20 +26,19 @@ def test_review_catalog(tmpdir):
     acar = aca.get_review_table()
     acar.run_aca_review()
     assert acar.messages == [
-        {'category': 'warning',
-         'idx': 2,
-         'text': 'Guide star imposter offset 2.6, limit 2.5 arcsec'},
-        {'category': 'critical', 'text': 'P2: 2.84 less than 3.0 for ER'},
-        {'category': 'critical', 'text':
-         'ER count of 9th (8.9 for -9.9C) mag guide stars 1.57 < 3.0'}]
+        {'text': 'Guide star imposter offset 2.6, limit 2.5 arcsec', 'category': 'warning',
+         'idx': 2},
+        {'text': 'P2: 2.84 less than 3.0 for ER', 'category': 'critical'},
+        {'text': 'ER count of 9th (8.9 for -9.9C) mag guide stars 1.91 < 3.0',
+         'category': 'critical'}]
```

Comment:
What changed in here?

Reply:
Using chandra_aca.star_probs.guide_count instead of the (previous) local implementation.

Reply:
Ah, so 1.57 -> 1.91 is the change and the rest is formatting.